samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -47,7 +47,7 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
   const afterCreate = useCallback(
     async (adapter: CallAdapter): Promise<CallAdapter> => {
       adapter.on('callEnded', () => {
-        onCallEnded();
+        // onCallEnded();
```

Comment:
Is there specific reason to add comment here?

Reply:
(this I assume is just for testing the ensure the composite's end call page is shown and not the sample's custom end call page)